compiler/codegen/codegen.cpp

```diff
@@ -1962,6 +1973,17 @@ void codegen(void) {
     }
   }
 
+  // Wrap calls to chosen functions from c library
+  if( llvmCodegen ) {
+#ifdef HAVE_LLVM
+    forv_Vec(FnSymbol, fn, gFnSymbols) {
+      if (fn->hasFlag(FLAG_EXTERN)) {
+        fn->cname = getClangBuiltinWrappedName(fn->cname);
```

Comment:
I think it'd be nicer to keep the conditional in here, since most of the time fn->cname does not change. Having the conditional here will make that more obvious to people reading the code.